package.json

```diff
@@ -89,8 +87,7 @@
     "vite-plugin-svgr": "^4.2.0",
     "vite-tsconfig-paths": "^4.2.3",
     "vitest": "^1.1.3",
-    "vitest-dom": "^0.1.1",
-    "workbox-window": "^7.0.0"
```

Comment:
~I don't remember if this one was needed for the service worker during build (the SW presented a lot of problems a few months ago when I moved from the old CRA service worker to VitePWA~

`vite-plugin-pwa` (which is the only package that could need this) already has `workbox-window` (\^7.0.0) as a dependency, so there's no use in explicitly declaring it here.

I'll keep an eye on this one during the next release, just in case.

Reply:
in the `yarn.lock` file at line 12607 the dependency

```
"workbox-window@npm:7.0.0, workbox-window@npm:^7.0.0":
  version: 7.0.0
  resolution: "workbox-window@npm:7.0.0"
  dependencies:
    "@types/trusted-types": "npm:^2.0.2"
    workbox-core: "npm:7.0.0"
  checksum: 10c0/8070fb80734a8543248ed6e3b186b8d3ee64419400a6d6d75f78809c44bf16321c5d9739bdf60ee62f631461ebb67a8a7dd78192504678d00bf2a9d3683f1324
  languageName: node
  linkType: hard

```

is there, so, assumptions is, nothing will be broken.

Reply:
that's correct!